vhpidirect/quickstart/logicvector/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+static const char HDL_LOGIC_STATE[] = { 'U', 'X', '0', '1', 'Z', 'W', 'L', 'H', '-'};
+
+char logic_vec[9];
+int logic_vec_size = 9;
+
+int getLogicVecSize(){
+  return logic_vec_size;
+}
+
+char* getLogicVec(){
+  printf("1D Array Logic Values [%d]:\n", logic_vec_size);
+  for(int i = 0; i < logic_vec_size; i++){
```

Comment:
Here, we are creating an array which has the same length as the number of types that are supported, plus we are giving each item the same value as the index. I would try to avoid `logic_vec[i] = i;`. Let's make the content be clearly independent from the index.

- Option 1: `logic_vec[i] = logic_vec_size-1-i;`.
- Option 2: split the vector in two, one with 4 elements and the other one with 5.

Regardless of changing the order/vectors, I think this is worth a comment. However, it can be included in the docs and not here. It's up to you!

Reply:
Small comment made in each getLogicVecA/B, in case someone grabs the code without looking at the docs for some reason. Documentation will highlight it.

Also, I set the values for VecA using HDL_* to make it clear that one is just setting a number, not actually a character